www/js/init.js

```diff
@@ -49,7 +49,7 @@ params['storeType'] = getBestAvailableStorageAPI();
 // The key prefix used by the settingsStore.js (see comment there for explanation), but we also need it below
 params['keyPrefix'] = 'kiwixjs-';
 params['hideActiveContentWarning'] = getSetting('hideActiveContentWarning') === true;
-params['showUIAnimations'] = getSetting('showUIAnimations') !== false;
+params['showUIAnimations'] = getSetting('showUIAnimations') !== true;
```

Comment:
This needs to be ` === true;`. The logic is that if the user has set it explicitly to `true`, then this will return `true`. But if it's any other value (`null`, '', `undefined`, `false`, 'false'), it will be `false`.